controllers/course.php

```diff
@@ -162,6 +162,11 @@ function index_action($active_id = 'false', $upload_message = false)
             $this->flash['messages'] = ['error' => $this->_('Die Datei konnte nicht erfolgreich hochgeladen werden.')];
         }
 
+        $this->mayWatchEpisodes = $perm->have_studip_perm('autor', $this->course_id);
```

Comment:
Huch, sollte dieser Commit das in diesem PR landen?

Reply:
heh, nein, du hast recht.